actiontext/lib/action_text/attachable.rb

```diff
@@ -64,7 +64,9 @@ def previewable_attachable?
     end
 
     def as_json(*)
-      super.merge(attachable_sgid: attachable_sgid)
+      super.tap do |json|
+        json.merge(attachable_sgid: attachable_sgid) if persisted?
+      end
```

Comment:
When I tested this locally, it looked like `as_json` includes `id: nil` when the record is not persisted. Instead of not including `attachable_sgid` at all, should it just be set to `nil`?

Reply:
@skipkayhil thanks a lot for your comment.
I just made the change so that we include `attachable_sgid` and set it to `nil`.